### DISCUSSION TITLE
Development Stalled , is qtile getting abandoned?

### DISCUSSION THREAD

**Discussion by v3ss0n** [qtile/qtile#4341] [General]
With release was almost a year ago , i think QTile development had burned out. Any thoughts adding other maintainers so it won't be stuck with inactive maintainer ?

**Comment by elParaguayo** [comment]
Have you even checked the commit history on the main branch? To say the developers are inactive is just plain wrong.

Yes, the last release was almost a year ago but that should not be confused with no development.

**Comment by v3ss0n** [comment]
Thanks , i am looking to move to QTile from KDE. Gonna try your discord.

**Reply by elParaguayo** [reply]
Sounds good. Feel free to ask questions if you ever need some help.

Hope you like it!

**Reply by v3ss0n** [reply]
One week in and i had moved everything there , i am liking  it a lot!

**Reply by elParaguayo** [reply]
Very glad to hear it.

**Reply by v3ss0n** [reply]
Keep up the great work.

**Reply by v3ss0n** [reply]
One small question : when can we see another release ?

**Reply by elParaguayo** [reply]
I'm really hoping it's soon. It's been much longer between releases than we had planned!

The main thing we're waiting for is just fixing the last couple of bugs in a major Wayland-related update. Once that's done, we'll push the release.

**Reply by v3ss0n** [reply]
thanks a lot , i will see if there are things i could help. being a pythonist , configuring  qtile is the best configuration experience of all time , no lua configs can beat this.
